src/common/snippets/include/snippets/op/loop.hpp

```diff
@@ -56,11 +56,8 @@ class LoopBegin : public LoopBase {
  * @param args vector of input values + LoopBegin, all values except for the LoopBegin are passed directly to output.
  * @param work_amount total number of evaluations to be processed by the loop
  * @param increment number of evaluations processed in one iteration of the loop.
- * @param apply_increment describes which data pointers attributed to the loop should be incremented on every iteration.
- * should be used when Loop is connected to Parameters and/or Results. If apply_increment[i] == true then i-th i/o data
- * pointer will be incremented by work_amount*data_size on every iteration.
- * @param ptr_increments specifies i/o pointer increment performed on every iteration. This is an alternative to
- * apply_increments, which enables more flexibility.
+ * @param is_incremented describes which data pointers attributed to the loop should be incremented on every iteration.
+ * @param ptr_increments specifies i/o pointer increment performed on every iteration if the following is_incremented[i] is true
```

Comment:
Actually it can be expressed. Need to revert this change - I will do that in follow up PRs

Reply:
I think that it can be useful for dynamic loops (for dynamic loop end emitter) when you need this port as `LoopPort` but it shouldn't be incremented.

Reply:
> I think that it might be useful for dynamic loops (for dynamic loop end emitter) when you need this port as `LoopPort` but it shouldn't be incremented.

Lets refactor in bounds of DynamicLoop enablement then.